### DISCUSSION TITLE
🌹 ROSE Bloom Grants • OVERVIEW & FAQ

### DISCUSSION THREAD

**Discussion by jcueto** [oasisprotocol/community-forum#13] [Grants Program]
![image](https://user-images.githubusercontent.com/5066732/106105783-6276ad00-60f9-11eb-8a08-6b07d598e61e.png)
-----
# GRANTS PROGRAM OVERVIEW
### Apply here: https://oasisfoundation.typeform.com/to/HtYql2aN
-----

_Note: This discussion thread provides an overview of the ROSE Bloom Grants Program. If you have any questions about the program, you can ask the Oasis Foundation team directly by posting a comment on this thread. We'll answer your questions as soon as possible and add common questions to the Frequently Asked Questions (FAQ) section of this post. We also encourage community members to share feedback and offer any suggestions for making the program even better. Thank you for your support!_

-----

The ROSE Bloom Grants Program is an entirely new grants program designed to accelerate the growth of the Oasis Network and build valuable new ecosystem tools. 

We’re excited to introduce a fully revamped grants program and the best way to receive funding for your project on the Oasis Network. Designed to help foster the growth of the Oasis ecosystem, the ROSE Bloom Grants program will help kickstart the development of projects in a range of key focus areas. 

The Bloom program is building on the success of the original grants and DevAccelerator program to ensure that the community has the resources needed to push the Oasis Network to new limits and strengthen its ability to support privacy-enabled DeFi and data tokenization.

The ROSE Bloom Grants program will focus on leveraging the following development environments:
- Oasis Labs' Parcel SDK: A privacy-first data governance SDK built by Oasis Labs to isolate and protect your most sensitive data without sacrificing the ability to use it.
- Second State’s Oasis Ethereum ParaTime: An Ethereum backwards compatible ParaTime with support for the entire Solidity toolkit. The ParaTime is nearly identical to Ethereum, but has better performance and lower gas fees.
- ParaTime SDK Development: ParaTimes are parallel computation environments that support smart contract execution on the Oasis Network. They can be built and run by anyone, and are highly customizable — allowing them to support a broad range of use cases and applications.

Support will take two main forms: bounties for specific projects that the Foundation sees as critical for the progress of the Oasis Network and more open-ended grants for new project ideas. 

If you are curious to learn about what grants have already been considered and approved, check out our [Github discussion threads for each project category](https://github.com/oasisprotocol/community/discussions/categories/grants-program). In these project-specific category threads, grant recipients will share updates on their projects and engage with questions and feedback from the community. You can also see what grant proposals were not approved, share new grant ideas, and comment on existing projects. 

-----

# FREQUENTLY ASKED QUESTIONS (FAQ)

### How do I apply for a grant?

Applying for a grant is easy. Just fill out the [application form](https://oasisfoundation.typeform.com/to/HtYql2aN) with the relevant information we are requesting, and we’ll notify you via email regarding the status of your application. 

The more specific and clear you are in your application, the better our team will be able to understand your project proposal.

-----

### Who is eligible to apply for a grant?

Eligibility to get approved for a grant is subject to KYC and your country of residence's accredited investor verification guidelines, as outlined in the official program rules (https://docsend.com/view/xykrayt6vb5tn7eh). 

Developers who are U.S. citizens or residents and who are not accredited investors may not be eligible to receive ROSE Bloom Grants. The grants are not and will not be offered or sold, directly or indirectly, to any person who is a resident, organized, or located in any country or territory subject to OFAC comprehensive sanctions programs from time to time, including Cuba, Crimea region of Ukraine, Democratic People’s Republic of Korea, Iran, Syria, any person found on the OFAC Specially Designated Nationals, Blocked Persons List, any other consolidated prohibited persons list as determined by any applicable governmental authority, or any persons who are organized in, located in, or residents of New York State.

-----

### How much money can I receive for my project?

Each proposal will be eligible to receive anywhere from **$5,000 to $50,000 USD** given out in **ROSE tokens**. Please specify the grant amount you are requesting in your application. The grant amount requested should match the scope of work for each project. 

-----

### What do you require for my project to get approved to receive a grant?

The Oasis Foundation team will assess your project proposal based on the following scoring categories: 
- Strength of the team, based on past projects, resume, GitHub, etc. - 1 (Very Weak) to 5 points (Very Strong) 
- Written quality of project proposal, based on details described, content, etc. - 1 (Very Low Quality) to 5 points (Very High Quality)
- Tech & product quality of project, based on technical implementation details and product vision, design overview, etc. - 1 (Very Low Quality) to 5 points (Very High Quality)
- Relevance to the success of the network, based on whether this is a product that could help us grow, an app that is in-demand among our community members, etc.- 1 (Not very relevant at all) to 5 points (Very Relevant)

Community interest and excitement generated will be a factor considered in the selection process, as an input into the “Relevance to the success of the network” scoring category. We’ll post project overviews on the Oasis GitHub discussion board to allow Oasis community members to share feedback, ask questions, and offer suggestions for each project. 

We also require you to provide specific milestones for assessing the progress of your project. Milestones should be decided based on the type of project you are developing and the scope of work for the project. In general, the types of things you can list as project milestones include specific development steps (e.g. initial deployment, debugging, obtaining first user, etc.), specific components of the application (e.g. sign-in flow, front-end user interface, database config, etc.), as well as key success metrics (e.g. app installs, number of transactions, number of users, etc.).

Depending on the size of your grant, the Oasis Foundation may disburse your grant in separate parts. Smaller grants may be disbursed in a single transaction. Larger grants may be disbursed in separate transactions at separate times, based on evidence of your continued progress building your project and meeting the milestones we agreed upon. 

Please note that we expect a commitment of at least around 3 months, including project development, launch, maintenance, etc. If you want to continue working on your project or developing extensions for it beyond this initial commitment, you can apply for additional grants in the future to fund your continued work. 

-----

### What happens after I apply for a grant?

Grant applications will be reviewed on an ongoing, rolling basis. In general, the application process will follow the timeline below: 
- Apply for a grant via our application form: https://oasisfoundation.typeform.com/to/HtYql2aN 
- Allow 1 to 2 weeks for the Oasis team to do a preliminary review of your application
- If your project idea looks promising, the Oasis team will post your project overview on the Oasis GitHub discussion board and notify you via email (applicants that are turned down at this preliminary stage will also be notified). This will allow the community to share feedback on your project proposal, ask questions, and offer you suggestions.
- Allow 1 to 2 weeks after your project gets posted on the Oasis GitHub discussion board and receives community feedback for the Oasis team to mark your project as ACCEPTED or TURNED DOWN. We will also notify you if your application is approved or turned down at this stage via email.
- If your grant application is accepted, the Oasis team will email you with next steps, including information about the KYC process.
The Oasis team will work closely with you to finalize the terms of your grant, and you can begin working on your project!

-----

### How long will it take to hear back about my application?

You should hear back about the status of your application within 1 to 2 weeks after you submit it. We'll be reviewing applications on an ongoing, rolling basis. 

-----

### How do I make suggestions or give feedback regarding the Oasis Network?

We love receiving feedback and suggestions from our community members!
- Join our Oasis Community Slack to view all of our support channels for developers: https://join.slack.com/t/oasiscommunity/shared_invite/zt-8a71w8i9-sVhe_5IG1hhb5Dgjl5xu2A 
- For logistical questions and feedback related to the ROSE Bloom Grants Program, post in the #grants-program-public channel.
- To learn more about the Oasis Network community of developers and supporters, join our Telegram: https://t.me/oasisprotocolcommunity 

-----

### For the full list of grant project categories, check out our Grants Program discussion threads: https://github.com/oasisprotocol/community/discussions/categories/grants-program

-----

**Comment by Olyno** [comment]
Hi 👋🏻 I have some questions about the grants program:
 * You offer us between $5 and $50k, we build our project, and that's it? No accountability to Oasis except the use of their blockchain?
 * What is the privacy of the information transmitted? I'm thinking mainly about the presentation of the idea, or KYC.
 * What are the risks in case of failure? It can happen that a project does not reach its end. Does this mean that Oasis has lost money, but nothing else is incurred, legally speaking?

*Sorry if my questions don't fly very high, I don't know this field very well 😅*
